gem/gem.py

```diff
@@ -616,6 +769,13 @@ def __new__(cls, summand, multiindex):
 
         return self
 
+    def latex(self):
+        index = ','.join([x.latex() for x in self.multiindex])
+        return r'\sum\limits_{{{0}}}\left \{{{1} \right \}}'.format(index, self.children[0].latex())
+
+    def _repr_latex_(self):
+        return r'${0}$'.format(self.latex())
+
```

Comment:
Sure. Will create a new branch to do the latex printing. Rendering is very slow though (compare to generating the latex string) so I guess it's unlikely to be useful for complicated forms.